### DISCUSSION TITLE
JSON Error: Syntax error in Httpie.php

### DISCUSSION THREAD

**Discussion by herbyxxx** [deployphp/deployer#3086] [Help needed]
- Deployer version: 7.0.0-rc.8
- Deployment OS: Windows
- PHP Version: 8.1.4

```
# php vendor\bin\deployer.phar deploy
task deploy:info
In Httpie.php line 181:

  JSON Error: Syntax error
```

The issue seems to be inside the `Configuration.php - load` function which tries to send a curl request to `http://localhost:PORT/load` which returns HTTP Code 411.

deploy.php
```
<?php
namespace Deployer;

require 'recipe/symfony.php';

// Config

set('repository', REPOSITORY);

add('shared_files', []);
add('shared_dirs', []);
add('writable_dirs', []);

set('ssh_multiplexing', false);

// Hosts

host(HOST)
    ->set('remote_user', USERNAME)
    ->set('deploy_path', PATH);

// Tasks

task('build', function () {
    cd('{{release_path}}');
    run('npm run build');
});

after('deploy:failed', 'deploy:unlock');

```
Connecting to the host via ssh works fine

**Comment by antonmedv** [comment]
Please, try to get this json out for debugging. Otherwise can’t help.

**Reply by thorau** [reply]
Have the same error. It is in the phar. Cannot get out the json for debugging.

**Reply by thorau** [reply]
It is resolved for me. Internally curl is used to acces http://localhost. I am behind a proxy and curl needs the no_proxy env var containing "localhost", otherwise it is reaching the proxy. The proxy's error message is html and that is why the mentioned error message is saying: JSON Syntax error.
